CoreFoundation/Locale.subproj/CFCalendar_Internal.h

```diff
@@ -19,7 +19,7 @@
 #include "CFDateComponents.h"
 #include "CFDateInterval.h"
 
-#if __has_include(<unicode/ucal.h>)
+#if __has_include(<unicode/ucal.h>) && !defined(__cplusplus)
```

Comment:
Per comment: please explain here.

Reply:
With C++ Interoperability enabled, all headers are imported as C++ headers. So they will all be imported in C++ mode. Even if they're not publically exposed. In other words, all headers in the CF module will be imported as C++ headers in a program that enables C++ interoperability. 

You're right, I think ForSwift may be the only place this is causing issues. To be honest, I know next to nothing about core foundation so, maybe this isn't the correct fix. Feel free to suggest an alternative. 

Anyway, does that answer your question? Would you also like me to put a comment in "CFCalendar_Internal.h" explaining this?

Reply:
@millenomi friendly ping. I'd really like to get this moving.

Reply:
My bad, I missed your ping back.

Reply:
No worries. I missed your comment the first time ;) Thanks!